docs/ref/contrib/admin/index.txt

```diff
@@ -2904,6 +2910,41 @@ The presence of the ``admin_password_reset`` named URL will cause a "forgotten
 your password?" link to appear on the default admin log-in page under the
 password box.
 
+.. _overriding-default-adminsite:
+
+Overriding the default ``AdminSite``
+------------------------------------
+
+.. versionadded:: 2.1
+
+You can override the default `django.contrib.admin.site` with your own:
```

Comment:
Please use double backticks, e.g. _django.contrib.admin.site_, _default_size_, and _AdminSite_.

Reply:
done